.circleci/generate_config_yml.py

```diff
@@ -131,10 +140,6 @@ def gen_build_workflows_tree():
     Header("Workflows"),
     Treegen(gen_build_workflows_tree, 0),
 
-    Header("Nightly tests"),
-    Listgen(binary_build_definitions.get_nightly_tests, 3),
-    File("workflows/workflows-nightly-uploads-header.yml"),
-    Listgen(binary_build_definitions.get_nightly_uploads, 3),
     File("workflows/workflows-s3-html.yml"),
```

Comment:
Maybe still want a header here?

Reply:
Adding the headers in is a little tricky since the nightly tests/uploads have been moved to the `binary_builds` workflow which is defined in a function outside of this function.

Also they don't really add much context to the jobs anyways.